GetStream/stream-chat-swift
# Add message preview with attachments in channel list

### 🔗 Issue Links
https://github.com/GetStream/ios-issues-tracking/issues/550

### 🎯 Goal
Add message attachment previews in the Channel List. This is a feature that is already present in the other SDKs. 

### 🎨 Showcase
The snapshots are a good way to see how message previews are rendered with attachments.

### 🧪 Manual Testing Notes
1. Open DemoApp
2. Open a Channel
3. Send a message with an attachment
4. Go to the Channel List
5. The attachment should be visible in the message preview

**Note**: It should be tested for Image, Video, File and Giphy attachment.

### ☑️ Contributor Checklist
